scripts/docker-image-builder.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
```

Comment:
for this mostly trusting you; i am a "linter" and I just checked that args are quoted!

Reply:
ACK, not a bash pro neither, but tested everything I could test I think, will see how it runs on the CI